source/computer/macos-setup.rst

```diff
@@ -29,40 +29,59 @@ macOS 配置指南
 Command Line Tools for Xcode
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Xcode 是 macOS 下的集成开发环境（IDE），类似于 Windows 下的 Microsoft Visual Studio。
-Command Line Tools for Xcode 是 Xcode 的一个子集，其包含了常用的命令行开发工具，
-比如 C 编译器、make、git 等，是安装很多其他软件的基础。
+`Xcode <https://developer.apple.com/cn/xcode/>`__ 是 macOS 下的集成开发环境（IDE），
+类似于 Windows 下的 `Microsoft Visual Studio <https://visualstudio.microsoft.com/>`__\ 。
+Command Line Tools for Xcode 是 Xcode 的一部分，其包含了常用的命令行开发工具，
+比如 C 编译器、\ ``make``\ 、\ ``git`` 等，是 macOS 下编程开发的必须软件。
 
-按下 :kbd:`Command+空格`\ ，搜索并打开“Terminal”。在 Terminal 中执行如下命令
-以安装 Command Line Tools for Xcode::
+按下 :kbd:`Command+空格`\ ，搜索“Terminal”并按下 :kbd:`Enter` 键以打开 Terminal 应用。
+在 Terminal 中执行如下命令以安装 Command Line Tools for Xcode::
 
    $ xcode-select --install
 
-在弹出的窗口点击“Install”即可。
+在弹出的窗口中点击“Install”即可。
 
-此处安装的 Command Line Tools 可能不是最新版，可以到“App Store”的“更新”中或
-“设置”的“软件更新”中查看是否有相关更新。如果有，则升级到最新版。
+此处安装的 Command Line Tools for Xcode 可能不是最新版。点击右上角的 Apple 图标，
```

Comment:
I don't have apple icon on the topright?

Reply:
Sorry, it should be left top. Do you have 
![image](https://user-images.githubusercontent.com/3974108/104693799-4118c880-56d8-11eb-8dcf-abf7195b1f6a.png)?

Reply:
yes, I have

Reply:
OK, need to make the change.